test/powershell/Modules/Microsoft.PowerShell.Utility/Invoke-Item.Tests.ps1

```diff
@@ -59,6 +59,11 @@ Describe "Invoke-Item basic tests" -Tags "CI" {
         }
         Get-Content $redirectFile -Raw | Should Match "usage: ping"
     }
+
+    It "Should invoke a folder without error" {
+        # can't validate that it actually opened, but no error should be returned
+        Invoke-Item -Path $PSHOME
```

Comment:
On windows we can validate it by using our COM adapter:
```
$s = New-Object -ComObject "Shell.Application"
$w = $s.Windows()

$before = $w.Count
Invoke-Item -Path $PSHOME
$after = $w.Count

$before + 1 | Should Be $after
$item = $w.Item($after - 1)
$item.LocationURL | Should Match ($PSHOME -replace '\\', '/')
## close the windows explorer
$item.Quit()
```
But no easy way to do the same on non-windows platforms.